baselines/bigquery-storage/test/gapic-big_query_storage-v1beta1.ts.baseline

```diff
@@ -41,7 +42,9 @@ export interface Callback {
 }
 
 export class Operation{
+    /* eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
Yaraaaaaaaarr.  This is a silly rule.  Could I trouble you for a PR to disable this rule in `gts`?  I thought I already had ...

Reply:
is it the same with `"node/no-empty-function": "off",`? in last PR for gts, I set this actually `"node/no-empty-function": "off",` but it complains about empty function. should I explicitely set `"@typescript-eslint/no-empty-function": "off"` instead? Thank you!

Reply:
Looks like both 😆   Yes please, let's disable this silliness.

Reply:
Got it, making the PR now, thanks for the catch! Please let me know when new release of gts is coming then I can update this PR :)

Reply:
PR sent out: https://github.com/google/gts/pull/467